build.fsx

```diff
@@ -135,7 +163,7 @@ Target "NuGet" (fun _ ->
             ReleaseNotes = String.concat " " release.Notes
             Tags = tags
             OutputPath = "bin"
-            ToolPath = nugetPath
+            ToolPath = "nuget.exe"
```

Comment:
just remove the toolpath parameter completely. FAKE will discover nuget.exe in the packages folder.